src/components/PremiumWarningDialog.tsx

```diff
@@ -0,0 +1,53 @@
+import { forwardRef } from 'react'
+import styled from 'styled-components'
+
+import { Dialog, DialogRef, Button, Icon } from '~/components/designSystem'
+import { useInternationalization } from '~/hooks/core/useInternationalization'
+import { theme } from '~/styles'
+
+export interface PremiumWarningDialogRef extends DialogRef {}
+
+export const PremiumWarningDialog = forwardRef<DialogRef>(({}, ref) => {
+  const { translate } = useInternationalization()
+
+  return (
+    <Dialog
+      ref={ref}
+      icon={
+        <BGIcon>
+          <Icon color="secondary" name="sparkles" />
+        </BGIcon>
+      }
+      title={translate('text_63b3155768489ee342482f4f')}
+      description={translate('text_63b3155768489ee342482f51')}
+      actions={({ closeDialog }) => (
+        <>
+          <Button variant="quaternary" onClick={closeDialog}>
+            {translate('text_62f50d26c989ab03196884ae')}
+          </Button>
+          <LinkTo href="mailto:xyz@yourapplicationdomain.com">
+            <Button>{translate('text_63b3155768489ee342482f55')}</Button>
+          </LinkTo>
+        </>
+      )}
+    />
+  )
+})
+
+const BGIcon = styled.div`
+  width: ${theme.spacing(10)};
```

Comment:
Hummm I don't think we should use spacing as width... 
Spacing are a specific system just for spacing. Width can be out of this system (and also let's say we change the system of spacing by putting it in base 2, this will f*** up the width based on it.
Just use the hardcoded value here